nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1488,6 +1493,31 @@
     patterns:
       - '_cython/_credentials/roots.pem'
 
+- module-name: 'gruut' # checksum: 19583930
+  data-files:
+    patterns:
+      - 'VERSION'
+
+  implicit-imports:
+    - depends:
+        - 'gruut_ipa'
+        - 'gruut_lang_en'
+
+- module-name: 'gruut_ipa' # checksum: 810a705a
+  data-files:
+    dirs:
+      - 'data'
+
+- module-name: 'gruut_lang_en' # checksum: 206e4c65
+  data-files:
+    dirs:
+      - 'espeak'
+      - 'g2p'
+      - 'pos'
+    patterns:
+      - 'VERSION'
+      - 'lexicon.db'
```

Comment:
The decision if a package is included is not impacted by configuration being present, the end user of Nuitka still can use it or not, this just makes it work in case he does.